SU2_PY/FSI_tools/FSIInterface.py

```diff
@@ -565,8 +565,12 @@ def interfaceMapping(self,FluidSolver, SolidSolver, FSI_config):
             # Note that the fluid solver is separated in more processors outside the python script
             # thus when, from a core, we request for the vertices on the interface, we only obtain
             # those in that core
-            GlobalIndex = FluidSolver.GetVertexGlobalIndex(self.fluidInterfaceIdentifier, iVertex)
-            posx, posy, posz = FluidSolver.GetInitialMeshCoord(self.fluidInterfaceIdentifier, iVertex)
+            GlobalIndex = FluidSolver.GetMarkerNodes(self.fluidInterfaceIdentifier, iVertex)
+            if self.nDim == 2:
+                posx, posy = FluidSolver.GetMarkerInitialCoordinates(self.fluidInterfaceIdentifier, iVertex)
```

Comment:
Is GetMarkerInitialCoordinates a new function?
Can the function return all coordinates (instead of this statement in another python script?

Reply:
It is an updated implementation of what used to be `GetInitialMeshCoord()` in python_wrapper_structure.cpp. `GetMarkerInitialCoordinates()` can be found in [CDriverBase.cpp](https://github.com/su2code/SU2/blob/feature_pysu2_DEF/SU2_CFD/src/drivers/CDriverBase.cpp#L474), and it only returns the components that are relevant to the number of dimensions in the problem.